features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/.eslintignore

```diff
@@ -3,6 +3,7 @@
 /coverage/**
 /source/test/**
 /source/src/app/components/Apis/Details/Security/**
+/source/src/app/components/Apis/Details/APIDefinition/**
```

Comment:
Why do we need to ignore this?
Shall we fix the eslint issues and remove this ignore line?

Reply:
Fixed https://github.com/wso2/carbon-apimgt/pull/6572/commits/547554b9a9c260c3115ce9cd4e74ad8008c3d85a